contracts/WildcardsQV.sol

```diff
@@ -61,11 +79,19 @@ contract WildcardsQV is Initializable {
     ////////////////////////////////////
     //////// SETUP CONTRACT////////////
     //// NOTE: Upgradable at the moment
-    function initialize(uint256 _votingInterval, ILoyaltyToken _addressOfLoyalyTokenContract, IWildCardToken _addressOfWildCardTokenContract, IWildCardSteward _addressOfWildCardStewardContract, uint _dragonCardId) public initializer {
+    function initialize(
+        uint256 _votingInterval,
+        ILoyaltyToken _addressOfLoyalyTokenContract,
+        IWildCardToken _addressOfWildCardTokenContract,
+        IWildCardSteward _addressOfWildCardStewardContract,
+        uint256 _dragonCardId
+    ) public initializer {
         admin = msg.sender;
         votingInterval = _votingInterval;
 
         proposalDeadline = now.add(_votingInterval);
+        currentHighestVoteCount = 0;
```

Comment:
These aren't super necessary since they are initialized to zero by default I'm sure, but was just doing it to be explicit. Since we do compare and use these variables before initializing them anywhere in the vote function.